staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -272,6 +272,13 @@ const (
 	// the 'nominalConcurrencyShares' field of the 'limited' section of a
 	// priority level.
 	ZeroLimitedNominalConcurrencyShares featuregate.Feature = "ZeroLimitedNominalConcurrencyShares"
+
+	// owner: @serathius
+	// beta: v1.30
+	//
+	// Allow watch create a separate watch RPC than watches opened to etcd.
+	// This prevents watch cache from being starved by watches to etcd.
+	SeparateCacheWatchRPC featuregate.Feature = "SeparateCacheWatchRPC"
```

Comment:
nit: put this between `RemainingItemCount` and `ServerSideApply` (keeping this vaguely alphabetical minimizes conflicts)

Reply:
Ups, think my other feature gates broke this  :P